api/src/database/createDatabase.ts

```diff
@@ -0,0 +1,24 @@
+import { logger } from '../logger';
+import { destroyClientAndExit, setupConnectionConfig, setupDbClient } from './setupClient';
+
+export const createDatabase = async () => {
+    logger.info(`Creating database ${setupConnectionConfig.database}.`);
+    await setupDbClient.raw(`CREATE DATABASE ${setupConnectionConfig.database}`)
+        .then(
```

Comment:
either `await` or `.then` but doing both is odd. I'd say you also want the style to be consistent throughout the code and `await` is probably nicer code so I'd vote `await` everywhere.